pkgs/development/libraries/libgda/default.nix

```diff
@@ -24,8 +25,8 @@ assert postgresSupport -> postgresql != null;
   hardeningDisable = [ "format" ];
 
   nativeBuildInputs = [ pkgconfig intltool itstool libxml2 gobject-introspection vala ];
-  buildInputs = with stdenv.lib; [ gtk3 openssl libgee ]
-    ++ optional (mysqlSupport) libmysqlclient
+  buildInputs = with stdenv.lib; [ gtk3 openssl libgee libmysqlclient]
+    # ++ optional (mysqlSupport) libmysqlclient
```

Comment:
I couldn't find any docs about `optional`. Is that something you can pass to `nix-build` via command-line?